extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -476,6 +477,22 @@ export abstract class AzureAuth implements vscode.Disposable {
 
 	//#region network functions
 
+	private async makeGetRequest(url: string, token: string): Promise<AxiosResponse<any>> {
+		const config: AxiosRequestConfig = {
+			headers: {
+				'Content-Type': 'application/json',
+				'Authorization': `Bearer ${token}`
+			},
+			validateStatus: () => true // Never throw
+		};
+
+		const response: AxiosResponse = await axios.get(url, config);
+		// ADAL is being deprecated so just ignoring these for now
```

Comment:
You removed the ADAL code, right? So does this still apply?

Reply:
This doesn't apply, I'll remove it